### ISSUE TITLE
Wont start on my mac

### ISSUE THREAD

**Issue by ryanramage** [datproject/dat-desktop#639]
Downloaded the build from https://github.com/datproject/dat-desktop/releases/download/v3.0.0/dat-desktop-3.0.0.dmg

Copied successfully into applications. When I ran from the launcher, it failed to start.

Running from cli, I get

```
 ⟦ℝ:Contents⟧ ⦖ ./MacOS/Dat\ Desktop
(electron) 'getName function' is deprecated and will be removed. Please use 'name property' instead.
Trace/BPT trap: 5
```

Hope that helps. Not a lot of time to debug, but will run another build if one is available.

**Comment by okdistribute** [comment]
Hey @ryanramage thanks for the bug report. Did you have an older version of dat desktop installed before? 

I'll take a look at this on my mac and see if I can debug.

**Comment by okdistribute** [comment]
Figured this out. I was able to get it to run on my machine when I changed the version of Node to `12.4`, and downgraded to electron `6`.. incoming on 3.0.1

**Comment by okdistribute** [comment]
Fixed in 3.0.1, https://github.com/datproject/dat-desktop/releases/tag/v3.0.1

**Comment by ryanramage** [comment]
@okdistribute the new release looks like it fixed it. Thanks!
